### DISCUSSION TITLE
Design adjustments possible?

### DISCUSSION THREAD

**Discussion by fkmuck1** [jonaswinkler/paperless-ng#1404] [Feature Requests]
My wife and I use Paperless-NG for several use cases. With a little ingenuity, the program offers a really wide range of uses.
Once we use it for invoices, once for documents and my wife has discovered that here she has a great opportunity to digitally save her recipes.
In order to meet the different requirements for keywords, groups, etc ... I have set up 3 separate Dockers. It all works wonderfully. But we would like there to be a possibility to customize the interface of Paperless. For example, to display your own logo on the start page or a header that makes it obvious that you are in the installation for "recipes" or "invoices". It is purely a matter of appearance and personal perception.
Would such an adjustment be possible?
Thanks for the great work so far and for a really great program!

**Comment by cjohnson46** [comment]
I really wish this would be implemented!  (Change logo, header and background colors, etc.)  I own a couple businesses and this would make it easier to see which docker the employees are using.  Even if we could get a small explanation of how we could do it ourselves without any changes to this project.  Please consider it!!
